File: Python/1040.py
# -*- coding: utf-8 -*-
valores = input().split()
n1, n2, n3, n4 = valores

n1 = float(n1)
n2 = float(n2)
n3 = float(n3)
n4 = float(n4)

nExame = 0.0
media = ((n1 * 2) + (n2 * 3) + (n3 * 4) + (n4 * 1)) / 10

print('Media: %.1f' % media)

if media >= 7.0:
    print("Aluno aprovado.")

if media < 5.0:
    print("Aluno reprovado.")

if media >= 5.0 and media <= 6.9:
    print("Aluno em exame.")
    nExame = float(input())
    print("Nota do exame: %.1f" % nExame)
    media = (media + nExame) / 2
    
    if media >= 5.0:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print('Media final: %.1f' % media)

File: C/1040.c
#include <stdio.h>
 
int main() {
    double n1, n2, n3, n4, nExame, media;
    scanf("%lf %lf %lf %lf", &n1, &n2, &n3, &n4);
    media = ((n1 * 2) + (n2 * 3) + (n3 * 4) + (n4 * 1)) / 10;
    printf("Media: %.1f\n", media);
    if (media >= 7.0){
        printf("Aluno aprovado.\n");
    }
    else if (media >= 5.0)
    {
        printf("Aluno em exame.\n");
        scanf("%lf", &nExame);
        printf("Nota do exame: %.1f\n", nExame);
        if (nExame + media / 2.0 > 5.0){
            printf("Aluno aprovado.\n");
        }
        else{
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n", (nExame + media) / 2.0);
    }
    else{
        printf("Aluno reprovado.\n");
    }
    return 0;
}